tests/test_types.py

```diff
@@ -2774,3 +2776,81 @@ class Model(BaseModel):
         {'loc': ('my_none_dict', 'a'), 'msg': 'value is not None', 'type': 'type_error.not_none'},
         {'loc': ('my_json_none',), 'msg': 'value is not None', 'type': 'type_error.not_none'},
     ]
+
+
+@pytest.mark.parametrize(
+    'value,result',
+    (
+        ('1996-01-22', date(1996, 1, 22)),
+        (date(1996, 1, 22), date(1996, 1, 22)),
+    ),
+)
+def test_past_date_validation_success(value, result):
+    class Model(BaseModel):
+        foo: PastDate
+
+    assert Model(foo=value).foo == result
+
+
+@pytest.mark.parametrize(
+    'value,expected',
+    (
+        (date.today(), str(date.today())),
+        (date.today() + timedelta(1), str(date.today() + timedelta(1))),
+        (datetime.today(), str(date.today())),
+        (datetime.today() + timedelta(1), str(date.today() + timedelta(1))),
+    ),
+)
+def test_past_date_validation_fails(value, expected):
```

Comment:
the second argument will not be required after the change requested above, but note for future reference: the second argument is not "expected" rather `date_str` or similar.

Reply:
Based on this comment, I'll assume the answer for my previous question is "no".

Also, thanks! :)